cognite/neat/rules/models/entities.py

```diff
@@ -615,3 +616,46 @@ def _generate_cdf_resource_list(v: Any) -> list[AssetEntity | RelationshipEntity
     AnyHttpUrl,
     PlainSerializer(lambda v: str(v), return_type=str, when_used="unless-none"),
 ]
+
+
+def load_value_type(
```

Comment:
maybe rename it to load_information_value_type to be consistent with naming below